src/mame/drivers/pofo.cpp

```diff
@@ -480,10 +496,10 @@ void portfolio_state::select_w(uint8_t data)
 
 	switch (data & 0x0f)
 	{
-	case 3: m_rom_b = CCM_A; break;
-	case 7: m_rom_b = CCM_B; break;
-	case 0: m_rom_b = ROM_APP; break;
-	case 2: m_rom_b = ROM_EXT; break;
+	case CCM_A:   m_rom_b = CCM_A;   break;
+	case CCM_B:   m_rom_b = CCM_B;   break;
+	case ROM_APP: m_rom_b = ROM_APP; break;
+	case ROM_EXT: m_rom_b = ROM_EXT; break;
```

Comment:
This is kind of silly now that you’re assigning the constant if the value equals the constant.  Does it really only stick if the low four bits are one of these four values?  What happens if one of the twelve other possible values is written?

Reply:
It's undocumented, and given that these are probably directly connected to various bus drivers, I'm not too keen on trying to stick garbage on these on real hardware, so that question unfortunately has no answer...

This is just keeping the existing behavior this file, while also allowing to read the value back which is necessary for the correct operation of some BIOS services.

Reply:
You could still make this code simpler and less error-prone.